test/verifiedTests/analysis/testMultiSpeciesModelling/testJoinModelsPairwiseFromList.m

```diff
@@ -0,0 +1,56 @@
+% The COBRAToolbox: testJoinModelsPairwiseFromList.m
+%
```

Comment:
Would it be possible, to keep the old functionality but only as a wrapper (i.e. call the new code from the old function files?) 
This would make the addition backward compatible.